src/main/java/microsoft/exchange/webservices/data/dns/DnsClient.java

```diff
@@ -43,6 +43,23 @@
 public class DnsClient {
 
   /**
+   * Set up the environment used to construct the DirContext.
+   *
+   * @param dnsServerAddress
+   * @return
+   */
+  static Hashtable<String, String> getEnv(String dnsServerAddress) {
```

Comment:
I need the mechanism to be testable, but it's not really external API so I didn't want to make it public. What do you suggest?